make/autoconf/build-performance.m4

```diff
@@ -243,8 +243,12 @@ AC_DEFUN([BPERF_SETUP_CCACHE_USAGE],
       fi
     fi
 
+    if $ALLOW_ABSOLUTE_PATHS_IN_OUTPUT;then
```

Comment:
Does this even build? Surely you need `if test ...` here?

Reply:
It works fine, I did test it.
```
make/autoconf/jdk-options.m4:    ALLOW_ABSOLUTE_PATHS_IN_OUTPUT="false"
make/autoconf/jdk-options.m4:    ALLOW_ABSOLUTE_PATHS_IN_OUTPUT="true"
make/autoconf/jdk-options.m4:    ALLOW_ABSOLUTE_PATHS_IN_OUTPUT="false"
make/autoconf/jdk-options.m4:    ALLOW_ABSOLUTE_PATHS_IN_OUTPUT="true"
```
I see other code is using the test:
```
make/autoconf/flags-cflags.m4:    if test "x$ALLOW_ABSOLUTE_PATHS_IN_OUTPUT" = "xfalse"; then
```
So I will change it.

Reply:
Yes please always use the existing style in the build files - we know that they work correctly everywhere, whereas other approaches may not.